### DISCUSSION TITLE
Getting error after scanning qr code

### DISCUSSION THREAD

**Discussion by JoshuaBarawa** [devlikeapro/waha#180] [Q&A] [answered]
Hi,
I keep getting the following error after scanning the QR code.
{
  "statusCode": 422,
  "message": "The session under FAILED status. Please try to restart it.",
  "error": "Unprocessable Entity"
}
Even after restarting, I keep getting the same error. Please help.

**Comment by allburov** [comment, marked as answer]
What version do you use?
If it's not the latest one - please try to update.
If it's the latest one - could you send dockef container logs please?

**Reply by allburov** [reply]
Update means pull a new image and restart container.
You can check your version by calling GET /api/version/

**Reply by JoshuaBarawa** [reply]
Do I use the same credentials I used last time? or do I need to pay for new credentials?

**Reply by allburov** [reply]
You need to get the latest credentials
